### PR TITLE
revised per AM Code review

### PR DESCRIPTION
**Pull request by sterlingann**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,39 +12,39 @@
 <body>
   <!-- Section 1: white portion/racoon -->
   <section>
-    <hr class="line"></hr>
-    <article id="racoonsection">
+    <hr class="line">
+    <article id="whatIsTopCoat" class="articlestyle1">
```
Comment (GarthDB) [inline index.html]: Instead of camelCase try `what-is-topcoat`

File: index.html [hunk 2]
```diff
@@ -12,39 +12,39 @@
...
       <!-- preliminary id incase we want to scroll -->
-      <hr class="racoonstyle">
-      <figure> <img src="racoonsvghere" width="400" height="400" /></a>
+      <hr class="whatIsTopcoatStyle">
```
Comment (GarthDB) [inline index.html]: What are the `hr`s going to be? The triangle?

Comment (sterlingann) [reply index.html]: yeah I think so.  thoughts?

File: index.html (hunk 1 above)
Comment (sterlingann) [reply index.html]: ok, will do

File: index.html (hunk 2 above)
Comment (GarthDB) [reply index.html]: I think we can get away without the `hr`s we should be able to use pseudo elements (`:before`, `:after`, etc.)

Comment (sterlingann) [reply index.html]: ok I will do that in the scss

Comment (GarthDB) [reply index.html]: 👍

File: index.html [hunk 3]
```diff
@@ -12,39 +12,36 @@
 <body>
   <!-- Section 1: white portion/racoon -->
   <section>
-    <hr class="line"></hr>
-    <article id="racoonsection">
+    <hr class="line">
+    <article id="what-is-topcoat" class="what-is-topcoat-style">
```
Comment (GarthDB) [inline index.html]: The `-style` is unnecessary on the class names.

Comment (sterlingann) [reply index.html]: ok, I will take off the styles now...  the id's will have the same naming convention as the class names

Comment (GarthDB) [reply index.html]: There's not real problem with that.

File: index.html [hunk 4]
```diff
@@ -12,39 +12,36 @@
 <body>
   <!-- Section 1: white portion/racoon -->
   <section>
-    <hr class="line"></hr>
-    <article id="racoonsection">
+    <hr class="line">
```
Comment (GarthDB) [inline index.html]: Go ahead and remove these `hr`s as well and I'm not sure we need the `section` and the `article`. I'd remove the `articles` and put the class and id on the parent `section`.

**Pull request merged**